admin/admin-service/src/main/java/io/mosip/admin/controller/AdminController.java

```diff
@@ -0,0 +1,34 @@
+package io.mosip.admin.controller;
+
+import io.mosip.admin.packetstatusupdater.dto.ApplicantVerficationDto;
+import io.mosip.admin.packetstatusupdater.util.AuditUtil;
+import io.mosip.admin.packetstatusupdater.util.EventEnum;
+import io.mosip.admin.service.AdminService;
+import io.mosip.kernel.core.http.ResponseWrapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.persistence.Access;
+
+@RestController
+public class AdminController {
```

Comment:
Should we rename this to a more specific name like ApplicantDetailsController ?

Reply:
taken care

Reply:
Resolved